build.sbt

```
@@ -17,6 +17,20 @@ ThisBuild / firtoolVersion := {
   }
 }
 
+// Previous versions are read from project/previous-versions.txt
+// If this file is empty or does not exist, no binary compatibility checking will be done
+// Add waivers to the directory defined by key `mimaFiltersDirectory` in files named: <since version>.backwards.excludes
+//   eg. unipublish/src/main/mima-filters/5.0.0.backwards.excludes
+val previousVersions = settingKey[Set[String]]("Previous versions for binary compatibility checking")
+ThisBuild / previousVersions := {
+  val file = new java.io.File("project/previous-versions.txt")
```

Nit: 
```suggestion
  val file = new java.io.File("project", "previous-versions.txt")
```

I am not good about this, but the `/` separators are probably best avoided.